tests/results/test_disability.py

```diff
@@ -3,10 +3,10 @@
 # import numpy as np
 # import pandas as pd
 #
-# from vivarium.testing_utilities import build_table, TestPopulation
```

Comment:
This entire file is commented out so I haven't really updated it thoroughly aside from replacing TestPopulation and Mortality with BasePopulation

Reply:
Can you check to see if there is a JIRA ticket to introduce tests for the DisabilityObserver or uncomment this file. If not, can you create one?

Reply:
Created one